### PR TITLE
feat(component): page header home

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Fixes https://todaydesign.atlassian.net/browse/UOM17-124

[Preview](https://deploy-preview-1633--pattern-lib-unimelb.netlify.app/?selectedKind=Today%2FHeader&selectedStory=Page%20Header%20Home&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] Make sure to do not repeat yourself (DRY)
- [ ] Snapshots tested
- [ ] A11y tested
- [ ] CSS utilises BEM naming convention and structure
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] Crossbrowser testing (IE11, Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [ ] My changes generate no new warnings
- [ ] I have added tests (e2e and unit) that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Added the new component to the index.js (Vue and Lib) export

# Reviewer Checklist

- [ ] Code is not repeated (DRY)
- [ ] ES6+ code is used where possible
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] CSS utilises BEM naming convention and structure
- [ ] Snapshots tested against a copy of dev snapshots to check UI changes are intended
- [ ] A11y tested
- [ ] Crossbrowser tested in at least IE11

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 98e044a9a4a53fa0ce4daf61348b491a15fc20d0

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/60a759d855d9c30007e815c7

**Review by lmounsey — APPROVED** [review]

**Comment by anormand** [comment]
### Success Criterion 1.4.11 - Non-text Contrast

The visual presentation of the following have a contrast ratio of at least 3:1 against adjacent color(s):

**User Interface Components**
Visual information required to identify user interface components and states, except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;

**Graphical Objects**
Parts of graphics required to understand the content, except when a particular presentation of graphics is essential to the information being conveyed.

**Problem - Arrow icon**
- The contrast ratio of the pathfinder arrow circle against the background is 2.1:1, which is too low

**Solution - Arrow icon**
- Make the circular icon lighter, e.g. #6698CC

**Comment by gepifanio** [comment]
@lmounsey would you mind checking the colour contrast for this one with Jacob?

**Comment by lmounsey** [comment]
> Success Criterion 1.4.11 - Non-text Contrast

Hi @anormand 

Just wanted to clarify this one - given the arrow is the graphic in this scenario, does it supersede the circle contrast with the background? As the white text/icon colour is compliant?

**Comment by gepifanio** [comment]
Merging this one to make it available for Andrew, however still following up with the accessibility fix/update.

@anormand @lmounsey

**Comment by jaredhenderson** [comment]
Ok by me, tough I think the transparent image could have had a couple of format options (eg; WebP)... even at 8 bit, the PNG is 0.5MB which is pretty hefty

**Comment by anormand** [comment]
@lmounsey this is a fair point. I was treating the button as a whole, in which case we need the 3:1 contrast ratio against the background.

But you look at the arrow icon on its own, in which case the contrast ratio would be sufficient.

[Understanding Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) supports this argument by using an example of a phone icon.

contrast-phone

> The phone icon is a simple shape within the orange (#E3660E) circle. The meaning can be understood from that icon alone, the background behind the circle is irrelevant. The orange background and the white icon have a contrast ration greater than 3:1, which passes.
> 
> The graphical object is the white phone icon.

We are using the arrow icon elsewhere, so its meaning is consistent. And if we make the circle lighter, the arrow will be harder to see.

In conclusion, I agree with @lmounsey that we can leave the icon as it is.
